controllers/comments.controller.js

```diff
@@ -0,0 +1,12 @@
+const { insertComment } = require('../models/comments.model')
+
+exports.postComment = (req, res, next) => {
+	const { article_id } = req.params
+	if (article_id && typeof req.body.body === 'string') {
+		insertComment(article_id, req.body)
+			.then(({ rows }) => {
+				res.status(201).send({ comment: rows[0] })
```

Comment:
This opens up a good debate - is it worth sending through to the model when we know we'll get nothing back, or should we send back a 404 at the controller without bothering to go to the model?

Personally I think this checking should be in the model. The model makes the decisions, the controller just sends back what it is given.